lib/foodcritic/api.rb

```diff
@@ -254,14 +254,16 @@ def match(node)
         line: pos["line"].to_i, column: pos["column"].to_i }
     end
 
+    def ast_cache(size = nil)
```

Comment:
Can you add YARD comments here including the introduced field with the next feature release version of Foodcritic (16.1) so people know when this came about.